src/NextGen/DonationForm/Models/DonationForm.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Give\NextGen\DonationForm\Models;
+
+use DateTime;
+use Give\Framework\FieldsAPI\Form;
+use Give\Framework\Models\Contracts\ModelCrud;
+use Give\Framework\Models\Contracts\ModelHasFactory;
+use Give\Framework\Models\Model;
+use Give\NextGen\DonationForm\Factories\DonationFormFactory;
+use Give\NextGen\DonationForm\Traits\DonationFormCRUD;
+use Give\NextGen\DonationForm\ValueObjects\DonationFormStatus;
+
+/**
+ * @unreleased
+ *
+ * @property int $id
+ * @property string $formTitle
+ * @property DateTime $createdAt
+ * @property DateTime $updatedAt
+ * @property DonationFormStatus $status
+ * @property array $settings
+ * @property string $blocksContent
+ * @property Form $schema
+ */
+class DonationForm extends Model implements ModelCrud, ModelHasFactory
+{
+    use DonationFormCRUD;
+
+    /**
+     * @inheritdoc
+     */
+    protected $properties = [
+        'id' => 'int',
+        'formTitle' => 'string',
```

Comment:
We can just call this property `title` since we are already in the context of a form. `$form->title`